themes/delphi/assets/css/pages/_team.scss

```diff
@@ -6,3 +6,17 @@
 .team-member > img {
     border-radius: 5px;
 }
+
+#team-header {
```

Comment:
is possible prefer a CSS class over a unique ID

Reply:
updated to class, thanks